### PR TITLE
Add ipv6 examples for network policy API

### LINKED ISSUE #86185: Document NetworkPolicy compatibility with IPv6
**What would you like to be added**:
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.17/#ipblock-v1-networking-k8s-io mentions an IPBlock with only IPv4 examples. Add a comment or example to clarify that IPv6 network blocks are (or possibly aren't) valid here.

**Why is this needed**: Cluster operators who use IPv6 networking want to understand how to isolate different workloads inside their cluster network (and without reading source code).
Logged against kubernetes/kubernetes rather than the website repo because this is generated documentation.

Kubernetes already has single-stack support for IPv6 clusters
KEP for dual-stack support is https://github.com/kubernetes/enhancements/blob/master/keps/sig-network/20180612-ipv4-ipv6-dual-stack.md

/sig docs
/sig network
/priority backlog

### PR DESCRIPTION
**Pull request by aojea**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design

/kind documentation

> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:

Add IPv6 examples to the network policy API comments/docs

**Which issue(s) this PR fixes**:

Fixes #86185

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by aojea** [comment]
/hold
I'll need some guidance to understand the process to publish the docs
/assign @bowei @sftim

**Comment by sftim** [comment]
If the upstream ([k/k](https://github.com/kubernetes/kubernetes)) change goes in, the website will get updated via the usual (and, uh, not-very-well-defined process). There's at least one update per minor release. Not sure about patchlevel releases.

**Comment by sftim** [comment]
/retest
?

**Comment by sftim** [comment]
/priority backlog
I think

**Comment by aojea** [comment]
/retest
flake https://github.com/kubernetes/kubernetes/issues/86318

**Comment by aojea** [comment]
/retest 
flake https://github.com/kubernetes/kubernetes/issues/85762

**Review by thockin — COMMENTED** [review]
Thanks!

/lgtm
/approve

**Comment by aojea** [comment]
/hold cancel
seems this is the right way :)
https://github.com/kubernetes/kubernetes/pull/86511#issuecomment-568221332

**Comment by aojea** [comment]
/retest
flake, serviceIP affinity issue

**Comment by aojea** [comment]
/label api-review
/assign @liggitt 
there is no API changes, just adding IPv6 examples for documentation, should I follow the whole API review process?

**Comment by liggitt** [comment]
> there is no API changes, just adding IPv6 examples for documentation, should I follow the whole API review process?

nope, no need for that, although if you want this doc to actually appear in API docs, you need to change the external types (in {networking,extensions}/v1beta1/types.go) and regenerate documentation (`make update`)

**Comment by aojea** [comment]
/hold
https://github.com/kubernetes/kubernetes/pull/86511#issuecomment-568640430

**Comment by aojea** [comment]
definitely seems that the nodePort thing is causing flakiness in the integration test suite, hit this errors again https://github.com/kubernetes/kubernetes/pull/86527, and this PR is totally unrelated

```
o/informers/factory.go:135: Unexpected watch close - watch lasted less than a second and no items received
F1224 09:46:37.490499  107628 controller.go:164] Unable to perform initial service nodePort check: unable to persist the updated port allocations: rpc error: code = Unavailable desc = transport is closing
FAIL	k8s.io/kubernetes/test/integration/master	481.045s
```

File: staging/src/k8s.io/api/networking/v1/types.go [hunk 1]
```diff
@@ -17,7 +17,7 @@ limitations under the License.
 package v1
 
 import (
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```
Comment (thockin) [inline staging/src/k8s.io/api/networking/v1/types.go]: why this?

Comment (aojea) [reply staging/src/k8s.io/api/networking/v1/types.go]: 🤦‍♂  visual studio autocomplete

**Comment by aojea** [comment]
/retest
flake [sig-storage] CSI Volumes [Driver: csi-hostpath] [Testpattern: Dynamic PV (default fs)] subPath should support existing single file [LinuxOnly]

**Comment by aojea** [comment]
/retest
flake [sig-storage] Downward API volume should update labels on modification [NodeConformance] [Conformance]

**Comment by aojea** [comment]
/retest
flake [sig-storage] Projected downwardAPI should provide container's memory limit [No

**Comment by aojea** [comment]
/retest

failed tests
[sig-api-machinery] CustomResourcePublishOpenAPI [Privileged:ClusterAdmin] works for multiple CRDs of same group but different versions [Conformance] expand_more	58s
[sig-api-machinery] CustomResourcePublishOpenAPI [Privileged:ClusterAdmin] works for CRD preserving unknown fields at the schema root [Conformance] expand_more	1m32s
[sig-api-machinery] CustomResourcePublishOpenAPI [Privileged:ClusterAdmin] removes definition from spec when one version gets changed to not be served [Conformance] expand_more	50s
[sig-api-machinery] CustomResourcePublishOpenAPI [Privileged:ClusterAdmin] works for multiple CRDs of same group and version but different kinds [Conformance] expand_more

**Comment by liggitt** [comment]
(flake tracked in https://github.com/kubernetes/kubernetes/issues/86418)

**Comment by liggitt** [comment]
/lgtm
per https://github.com/kubernetes/kubernetes/pull/86511#pullrequestreview-336024163

**Pull request merged**